### DISCUSSION TITLE
華為手機問答

### DISCUSSION THREAD

**Discussion by uwje** [1101-GS4719-C/Python-Community-GS4719-C#1610] [科技新知]
中美貿易戰僵持許久，導致美國限制中國進口特定產品，如:光刻機、稀有金屬與礦物、先進技術和生產線，外國公司也漸漸的把生產線轉移至勞動力也較為低廉的東南亞與印度。尤其華為這間公司，被針對地尤其嚴重，前幾年許多國家紛紛抵制華為建設的5G網路，導致華為每況愈下。
在此背景下，在前段時間，華為卻發布了宣稱使用先進晶片製造的新5G手機，但事實證明確是假的，他們不但沒有能力製造出先進晶片，也沒設計出與EUV實力相當的先進晶片光刻機。
我認為中國並未生產與現今潮流相同的晶片，但卻有讓人感覺到對中國的封鎖似乎並沒有非常有效。
參考連結:
https://technews.tw/2023/08/30/smics-5nm-process-breakthrough-foundry-huawei-kirin-9000s/

**Comment by Yanxxx7979** [comment]
同樣認同對於華為的封鎖所造成的影響可能會愈來愈小。一方面中國對人才的網集及培養科研人才的能力的確是不可小覷；另一方面，文中有提到，他們的晶片製造仍然是使用了部分已經對他們進行封鎖的技術，而這是否代表被封鎖的技術仍然有可能透過第三方國家流入中國呢？
